src/rules/armTemplate.ts

```diff
@@ -203,14 +151,17 @@ export class ARMTemplateRule implements BaseRule<ARMTarget> {
   async sendRequest(
     target: ARMTarget,
     resource: ARMResource,
-    evaluation: RequestEvaluation
+    request: RequestEvaluationObject
```

Comment:
nit: adding `Object` to the end of interface/class names usually doesn't add value or clarity & ends up being extraneous. We might review some naming